doc/jvm/xgboost4j_spark_tutorial.rst

```diff
@@ -286,16 +286,22 @@ Then we can load this model with single node Python XGBoost:
   bst = xgb.Booster({'nthread': 4})
   bst.load_model(nativeModelPath)
 
+When interacting with other language bindings, XGBoost also supports saving-models-to and loading-models-from file systems other than the local one:
+
+* You can build XGBoost4J-Spark with the steps described in `here <https://xgboost.readthedocs.io/en/latest/jvm/index.html#installation-from-source>`_, but turning `USE_HDFS <https://github.com/dmlc/xgboost/blob/e939192978a0c152ad7b49b744630e99d54cffa8/jvm-packages/create_jni.py#L18>`_ (or USE_S3, etc. in the same place) switch on. With this approach, you can reuse the above code example by replacing "nativeModelPath" with a HDFS path.
+
+However, if you build with USE_HDFS, etc. you have to ensure that the involved shared object file, e.g. libhdfs.so, is put in the LIBRARY_PATH of your cluster. To avoid the complicated cluster environment configuration, we recommend download file manually from HDFS and load with the pre-built version of XGBoost.
```

Comment:
What do you mean by "download file manually from HDFS"? Do you mean that the user should stick with native paths?

Reply:
I see what you mean here. I think you should also XGBoost4J needs USE_HDFS=ON to run `xgbClassificationModel.nativeBooster.saveModel()`. So USE_HDFS=ON is needed for both XGBoost4J-Spark and Python sessions.

Reply:
yes, that's what the above two paragraphs for `With this approach, you can reuse the above code example by replacing "nativeModelPath" with a HDFS path.`

Reply:
I'm confused about the recommendated action, since running
```
local_file_path = download_hdfs(hdfs_path)
xgb.load(local_file_path)
```
won't solve the problem of XGBoost4J-Spark unable to save to HDFS to begin with.

Reply:
I added description about how to save a pure booster to HDFS